app/models/condition.rb

```diff
@@ -0,0 +1,7 @@
+class Condition < ApplicationRecord
+  has_paper_trail
```

Comment:
This is fine, I think.

This means all changes will be recorded in the version table. They will be scoped to the Condition model so shouldn't interfere with any other draft/live features. If that's not useful, we can delete/ignore them.

The draft/live work has add two new routes `/draft` and `/live`, which the runner uses. These add pages into the main forms json so provide a complete description of a form. We'll need to include the conditions in these routes. When you get to that bit give us a shout if you need help.

Reply:
Do you think we should start adding conditions to the live form jsons as part of this PR, or save it for further down the line?